Create ui-c8y-1023-30-1-ui-remove-server-error-occurred-alert-message-when-user-has-limited.md (#4215)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-30-1-ui-remove-server-error-occurred-alert-message-when-user-has-limited.md

---
date: ""

> title: remove server error alert and hide alarms view when user has limited access to alarms

Reviewer: ```suggestion
title: Removed misleading error alerts and made the alarms view unavailable for users without permission
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63546
version: 1023.30.1
---

> When users with limited alarm permission viewed alarms, repeated "server error occurred" alerts appeared, hiding the actual permission message. The misleading error alerts have been removed, and the alarms view is now unavailable for users without alarm permission, showing a clear "You don't have permission to view alarms" message instead.

Reviewer: @DawidSAG What does limited alarm permission mean? Isn´t it "without alarm permission"?

Reviewer 2: You're right, I've updated it

> When users with limited alarm permissions viewed alarms, repeated "server error occurred" alerts would appear, hiding the actual permission message underneath. The misleading error alerts have been removed, and the alarms view is now hidden for users without alarm permissions, showing a clear "You don't have permission to view alarms" message in the widget instead.

Reviewer: ```suggestion
When users with limited alarm permission viewed alarms, repeated "server error occurred" alerts appeared, hiding the actual permission message. The misleading error alerts have been removed, and the alarms view is now unavailable for users without alarm permission, showing a clear "You don't have permission to view alarms" message instead.
```
